### PR TITLE
Add image processor to coordinator

### PR DESCRIPTION
**Pull request by twlogger**

### CODE REVIEW

**Comment by digitalcarp** [comment]
Does this code work for video? Or is it trivial to change to video processing.

**Comment by digitalcarp** [comment]
Could you move the ImageProcessor class to its own file and do like `from package.image_processor import ImageProcessor`

**Comment by twlogger** [comment]
How does the webcam hook up thru python? I imagine you can grab 10 frames per second and pass them in as numpy arrays which is what the ImageProcessor works with.

**Comment by digitalcarp** [comment]
https://opencv-python-tutroals.readthedocs.io/en/latest/py_tutorials/py_gui/py_video_display/py_video_display.html#capture-video-from-camera

Something like that?

**Comment by twlogger** [comment]
then it should be trivial to get this working with the real webcam data

File: Coordinator/package/coordinator.py [hunk 1]
```diff
@@ -1,10 +1,308 @@
...
+
 class Coordinator:
     def __init__(self, road):
         self.road = road
-    
+        self.processor = ImageProcessor()
+
     def update(self):
+        # Grab the frame which needs to be processed
+        image = mpimg.imread('overheadreal.jpg')
```
Comment (digitalcarp) [inline Coordinator/package/coordinator.py]: Instead of matplotlib image read can you try implementing with OpenCV image read `cv2.imread()`

**Comment by digitalcarp** [comment]
Actually I'm just gonna merge this rn and you can change stuff afterwards

**Pull request merged**